ci/gpu/build.sh

```diff
@@ -52,15 +52,15 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c rapidsai/label/x
       "lapack" \
       "cmake==3.14.3" \
       "umap-learn" \
-      "protobuf >=3.4.1,<4.0.0" \
+      "protobuf>=3.4.1,<4.0.0" \
       "nccl>=2.5" \
-      "dask>=2.8.0" \
-      "distributed>=2.8.0" \
+      "dask>=2.12.0" \
+      "distributed>=2.12.0" \
       "dask-cudf=${MINOR_VERSION}" \
       "dask-cuda=${MINOR_VERSION}" \
       "ucx-py=${MINOR_VERSION}" \
       "statsmodels" \
-      "rapids-xgboost>=0.13" \
+      "xgboost====1.0.2dev.rapidsai0.13" \
```

Comment:
Please leave the original but update it to use the following:

```
rapids-xgboost=${MINOR_VERSION} \
```

This is a version matched meta-pkg that we control that will give the correct xgboost version for the current rapids version. Hard coding to the version you have now will not work well past this PR.

Reply:
I think this is a 0.13=>0.14 merge change, not part of this PR. But it should be fixed now that the meta package is updated.